### PR TITLE
Add glean client_info descriptions

### PR DESCRIPTION
**Pull request by wlach**
* Add glean client info descriptions to base glean schema
* Add a simple script to keep these up to date

Checklist for reviewer:

- [ ] Commits should reference a bug or github issue, if relevant (if a bug is referenced, the pull request should include the bug number in the title)
- [ ] Scan the PR and verify that no changes (particularly to `.circleci/config.yml`) will cause environment variables (particularly credentials) to be exposed in test logs
- [ ] If the PR comes from a fork, trigger the `integration` CI test by pushing this revision [as discussed in the README](https://github.com/mozilla-services/mozilla-pipeline-schemas#packaging-and-integration-tests-optional) and review the report posted in the comments.

For glean changes:
- [x] Update `include/glean/CHANGELOG.md`

### CODE REVIEW

File: templates/include/glean/client_info.1.schema.json
Comment (wlach) [inline templates/include/glean/client_info.1.schema.json:53]: This appears not to be "documented" in the Glean SDK. @Dexterp37 is this an oversight?

**Comment by wlach** [comment]
After having gone to the trouble of writing this PR, I found out that the Glean SDK keeps its own copy of this file here for validation purposes:

https://github.com/mozilla/glean/blob/main/glean.1.schema.json

It is kept in sync with this script:

https://github.com/mozilla/glean/blob/main/bin/update-schema.sh

Longer term, I wonder if we should invert this relationship: keep/generate the schema inside glean itself, and then copy it over into this repository as needed.

**Review by fbertsch — COMMENTED** [review]
Can we validate that the descriptions are up-to-date in ci, [like we do here](https://github.com/mozilla/bigquery-etl/blob/master/.circleci/config.yml#L109)?

**Comment by wlach** [comment]
> Can we validate that the descriptions are up-to-date in ci, [like we do here](https://github.com/mozilla/bigquery-etl/blob/master/.circleci/config.yml#L109)?

I would feel a bit bad doing that, because it could cause PRs to fail for no fault of the person writing them. I think a better solution (long term) is what I outlined above: moving the schema generation (and the schema itself) to glean.

**Review by fbertsch — APPROVED** [review]
@wlach I'll leave it up to you then - for now this change seems appropriate.

File: templates/include/glean/client_info.1.schema.json
Comment (Dexterp37) [reply templates/include/glean/client_info.1.schema.json:54]: You can find the docs for that [here](https://mozilla.github.io/glean/book/user/pings/index.html#the-client_info-section).

File: scripts/extract_glean_client_info_descriptions
Comment (Dexterp37) [inline scripts/extract_glean_client_info_descriptions:1]: This file is missing the `.py` suffix and the license header . Any chance you could fix this before merging?

**Review by Dexterp37 — APPROVED** [review]

File: scripts/extract_glean_client_info_descriptions
Comment (wlach) [reply scripts/extract_glean_client_info_descriptions:1]: I'm inclined to leave this as-is:

1. The other python scripts don't have .py extensions
2. We don't generally add MPL license headers for internal tooling like this, as we consider the top-level license sufficient context. See e.g. mozilla/gcp-ingestion#777

Comment (Dexterp37) [reply scripts/extract_glean_client_info_descriptions:1]: Oh, TIL. Cool, go ahead then!

File: templates/include/glean/client_info.1.schema.json
Comment (wlach) [reply templates/include/glean/client_info.1.schema.json:54]: Got it. Should this be added here:

https://github.com/mozilla/glean/blob/ccf91a74dc0d95e5b62df2452187157e882f1907/glean-core/metrics.yaml#L32

?

Comment (Dexterp37) [reply templates/include/glean/client_info.1.schema.json:54]: Oh right, you're right.

**Pull request merged**
